addons/crm/views/crm_lead_views.xml

```diff
@@ -451,7 +451,7 @@
                                 'default_company_name': not commercial_partner_id and type == 'opportunity' and partner_name,
                                 'default_name': contact_name or partner_name,
                                 'default_is_company': False,
-                                'default_type': 'contact',
+                                'default_type': 'c',
```

Comment:
Why did you change type?

This `default_type` value belongs to the type field of `res.partnet` not  `l10n_in.pan.entity` .

The fix should be applied to the `l10n_in` module, not in CRM.